include/RE/H/HUDNotifications.h

```diff
@@ -12,29 +15,19 @@ namespace RE
 	public:
 		inline static constexpr auto RTTI = RTTI_HUDNotifications;
 
-		struct UnkData1
+		struct Notification
 		{
```

Comment:
```suggestion
		{
		public:
		// members
```

Reply:
I changed it back to `struct` its public by default :)

Reply:
I am aware that the default access specifier for `struct` is public. I would still like you to make this change for consistency across the code base.